extensions/wikia/UserLogin/js/UserSignupAjaxValidation.js

```diff
@@ -103,10 +104,6 @@
 			}
 		}
 
-		if (this.captchaField && this.checkFieldEmpty(this.wikiaForm.inputs[this.captchaField])) {
```

Comment:
Grecaptcha doesn't give us an event when the user completes the captcha, so we can't include it with ajax validation. :(